components/butterfly-test/src/lib.rs

```diff
@@ -165,12 +166,10 @@ impl SwimNet {
                 if pos == x_pos {
                     continue;
                 }
-                let server_b = self.members.get(x_pos).unwrap();
-                to_mesh.push(member_from_server(server_b))
+                to_mesh.push(member_from_server(&self.members[x_pos]))
             }
-            let server_a = self.members.get(pos).unwrap();
             for server_b in to_mesh.into_iter() {
-                server_a.insert_member(server_b, Health::Alive);
+                self.members[pos].insert_member(server_b, Health::Alive);
```

Comment:
Changing `members.get(foo).unwrap()` → `members[foo]` is totally mechanical, but once I did that, the need for the `let` statements seemed to go away, so I further simplified.